Packs/Zscaler/Integrations/Zscaler/Zscaler_description.md

```diff
@@ -1 +0,0 @@
-[View Integration Documentation](https://xsoar.pan.dev/docs/reference/integrations/zscaler)
```

Comment:
Is this removed by mistake?

Reply:
Hello JasBeilin, thank you for your answer !

It was indeed a mistake, I made another commit to revert this, please let me know if it wasn't the correct way to do that.

Regards,
Julian